apps/etterlatte-vedtaksvurdering/src/main/kotlin/no/nav/etterlatte/vedtaksvurdering/VedtakBehandlingService.kt

```diff
@@ -404,11 +404,18 @@ class VedtakBehandlingService(
     suspend fun oppdaterSamordningsmelding(
         samordningmelding: OppdaterSamordningsmelding,
         brukerTokenInfo: BrukerTokenInfo,
+        sakId: SakId,
     ) {
         repository.lagreManuellBehandlingSamordningsmelding(samordningmelding, brukerTokenInfo)
 
         try {
-            samordningsKlient.oppdaterSamordningsmelding(samordningmelding, brukerTokenInfo)
+            val meldingFinnes = samordningsKlient.oppdaterSamordningsmelding(samordningmelding, brukerTokenInfo)
+            if (meldingFinnes) { // TODO bør vi ha ytterlige sjekker før vi samordner?
```

Comment:
Hvordan begrense til de sakene vi vet om? og kan vi basere oss på HttpCode.Conflict i alle tilfeller

Reply:
Bør vi og sjekke response.getBody inneholder

```
"exceptionType":"ALLEREDE_REGISTRERT_ELLER_UTENFOR_FRIST"
```

Reply:
Er ikke det en sjekk som er mulig å gjøre direkte på dataene vi får og som presenteres for saksbehandler, slik at man evnt bare kan klikke direkte på noe á "samordning ferdig". Evnt lage en lytter i vedtaksvurdering som hver dag sjekker vedtak som står til behandling og har gjort det i > 6 uker, og kan gjøre overstyre resten av handlingsrekka.

Reply:
Problemet er at vi ikke mottar data for saker som er svart utenfor fristen (det undersøkes). Når saksbehandler forsøker å overstyrer får vi feil med melding (Se PR).

Så problemet er at vi ikke kan overstyre. Og vi mottar heller ingen data. Dette er den enkleste fiksen vi kom på uten å involvere større endringer..